engine/src/main/scala/org/scalarules/dsl/core/glossaries/Glossary.scala

```diff
@@ -11,7 +11,7 @@ import scala.language.experimental.macros
   * Utility base class for collecting and namespacing `Fact`s. You can extend this class, define facts in it and receive a
   * utility collection of all facts declared in your class.
   */
-class MacroGlossary {
+class Glossary {
   def defineFact[A](): SingularFact[A] = macro FactMacros.defineFactMacroImpl[A]
   def defineFact[A](description: String = "No description"): SingularFact[A] = macro FactMacros.defineFactMacroWithDescriptionImpl[A]
```

Comment:
Why do we give a default for the description parameter? If none is given, I assume the no-args method is called, making this default pointless